src/app/(app)/missing-shoes/actions/get-all-missing-shoes.ts

```diff
@@ -0,0 +1,28 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+
+export const getAllMissingShoes = async () => {
+  const supabase = await getSupabaseClient();
+
+  const { data: missingShoesData, error } = await supabase
```

Comment:
```suggestion
  const { data: missingShoes, error } = await supabase
```
I don't like the variables with the suffix 'Data' in them

Reply:
I have fixed this and I will take care about this in future